schema-templates/basic-auth.sql

```diff
@@ -0,0 +1,72 @@
+begin;
+
+create schema if not exists basic_auth;
+set search_path to basic_auth;
```

Comment:
Sure, I can switch to creating the schema and prefixing the other names with `basic_auth.`. My thought with this file (and others which can go into the schema-templates directory) is that people can apply them to their database to add common subsystems. This would be a file to run and then have full featured auth if that's possible. Minus the email sending part which would have to connect to this in a way I haven't thought through.